File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        set<int> setNoDup(nums.begin(), nums.end());
        vector<int> vectNoDup(setNoDup.begin(), setNoDup.end());
        sort(vectNoDup.begin(), vectNoDup.end(), [&nums](const auto& a, const auto& b){
            return count(nums.begin(), nums.end(), a) > count(nums.begin(), nums.end(), b);
        });

        vector<int> ans;

        for(int i =0; i < k; i++)
        {
            ans.push_back(vectNoDup.at(i));
        }

        return ans;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = maxCount = 0
        for num in nums:
            if num == 1:
                count += 1
            else:
                count = 0
            maxCount = max(count, maxCount)
            
        return maxCount
